spec/models/maneuver_participant_spec.rb

```diff
@@ -6,47 +6,47 @@
       expect(record.score).to be(50)
     end
 
-    it 'should go down by 10 for each reverse direction' do
+    it 'lowers by 10 for each reverse direction' do
       expect { record.update reversed_direction: 1 }
         .to change { record.score }
         .by(-10)
     end
 
-    it 'should go down by 25 when speed target is not reached' do
+    it 'lowers by 25 when speed target is not reached' do
       record.maneuver.update! speed_target: 25
       expect { record.update speed_achieved: false }
         .to change { record.score }
         .by(-25)
     end
 
-    it 'should go down by 25 when made additional stops' do
+    it 'lowers by 25 when made additional stops' do
       record.maneuver.update! counts_additional_stops: true
       expect { record.update made_additional_stops: true }
         .to change { record.score }
         .by(-25)
     end
 
-    it 'should go down by 50 when not completed as designed' do
+    it 'lowers by 50 when not completed as designed' do
       expect { record.update completed_as_designed: false }
         .to change { record.score }
         .by(-50)
     end
 
-    it 'should go down by the sum of each point value multiplied by count' do
+    it 'lowers by the sum of each point value multiplied by count' do
       expect { record.update obstacles_hit: { 1 => 1 } }
```

Comment:
So this test doesn't make it clear how the `1 => 1` data structure translates into a point deduction. It'd be nice to construct it incrementally with descriptive variable names, just to break it down a bit (same for the following tests).